### PR TITLE
Link to Umbraco backoffice site from CMS

### PR DESCRIPTION
**Pull request by sofietoft**
I've created a link to the new Umbraco Backoffice site from the article about the Backoffice UI API docs in the CMS docs.

### CODE REVIEW

File: 10/umbraco-cms/extending/backoffice-ui-api-documentation.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ versionFrom: 10.0.0
 
 A library of API Reference documentation is auto-generated from the comments within the Umbraco Source Code.
 
+{% hint style="info" %}
+A new backoffice for Umbraco CMS is currently being worked on. It is planned for release with Umbraco 13 i December of 2023.
```
Comment (eshanrnh) [inline 10/umbraco-cms/extending/backoffice-ui-api-documentation.md:10]:
```suggestion
A new backoffice for Umbraco CMS is currently being worked upon. It is planned to release with Umbraco version 13 in December 2023.
```

File: 10/umbraco-cms/extending/ui-library.md [hunk 2]
```diff
@@ -5,12 +5,7 @@ description: A guide for getting started working with the Umbraco UI Library
 # UI Library
 
 {% hint style="info" %}
-Please be aware that the topic covered in this document is currently only available in the following versions of Umbraco:
-
-* The latest version of Umbraco 10.
-* The Release Candidate (RC) for Umbraco 11.
-
-The new UI Library is currently _opt-in_ and something we recommend Backoffice and package developers start getting familiar with.
+The UI Library is currently _opt-in_ and something we recommend Backoffice and package developers start getting familiar with
```
Comment (eshanrnh) [inline 10/umbraco-cms/extending/ui-library.md:8]:
```suggestion
The UI Library is currently _opt-in_. We recommend Backoffice and package developers start getting familiar with it.
```

File: 11/umbraco-cms/extending/backoffice-ui-api-documentation.md [hunk 3]
```diff
@@ -2,6 +2,14 @@
 
 A library of API Reference documentation is auto-generated from the comments within the Umbraco Source Code.
 
+{% hint style="info" %}
+A new backoffice for Umbraco CMS is currently being worked on. It is planned for release with Umbraco 13 i December of 2023.
```
Comment (eshanrnh) [inline 11/umbraco-cms/extending/backoffice-ui-api-documentation.md:6]:
```suggestion
A new backoffice for Umbraco CMS is currently being worked upon. It is planned to release with Umbraco version 13 in December 2023.
```

File: 11/umbraco-cms/extending/backoffice-ui-api-documentation.md [hunk 4]
```diff
@@ -2,6 +2,14 @@
...
+
+We have gathered all information and resources about the project in one place, giving you a full overview of what we're working on. You will also be able to find information about how to get involved in the project.
+
+You can find it all on [docs.umbraco.com/umbraco-backoffice](https://docs.umbraco.com/umbraco-backoffice).
```
Comment (eshanrnh) [inline 11/umbraco-cms/extending/backoffice-ui-api-documentation.md:10]:
```suggestion
You can find it in the [The new Umbraco Backoffice](https://docs.umbraco.com/umbraco-backoffice) article.
```

File: 11/umbraco-cms/extending/ui-library.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ description: "A guide for getting started working with the Umbraco UI Library"
 # UI Library
 
 {% hint style="info" %}
-The UI Library is currently _opt-in_ and something we recommend Backoffice and package developers start getting familiar with.
+The UI Library is currently _opt-in_ and something we recommend Backoffice and package developers start getting familiar with
```
Comment (eshanrnh) [inline 11/umbraco-cms/extending/ui-library.md:8]:
```suggestion
The UI Library is currently _opt-in_. We recommend Backoffice and package developers start getting familiar with it.
```

File: 10/umbraco-cms/extending/backoffice-ui-api-documentation.md [hunk 6]
```diff
@@ -6,6 +6,14 @@ versionFrom: 10.0.0
...
+
+We have gathered all information and resources about the project in one place, giving you a full overview of what we're working on. You will also be able to find information about how to get involved in the project.
+
+You can find it all on [docs.umbraco.com/umbraco-backoffice](https://docs.umbraco.com/umbraco-backoffice).
```
[the remark this answers is not in the document]
Comment (sofietoft) [reply 10/umbraco-cms/extending/backoffice-ui-api-documentation.md:14]: It's actually not just one article, but an entire site.

**Pull request merged**